lpa-submissions-e2e-tests/cypress/integration/accuracyAppellantsSubmission.feature

```diff
@@ -0,0 +1,70 @@
+Feature: Accuracy of the Apellants Submission
+
+  As a beta LPAI want to respond whether the information from the appellant accurately reflects the original planning application
+  so that the Planning Inspectorate can make a decision.
+
+  Scenario: AC01 Access Review the accuracy of the appellant's submission page
+    Given the user is on the Task List page
+    When the user selects the link 'Review the accuracy of the appellant's submission'
```

Comment:
Step definition missing for this step.

Reply:
This is on line 26 of the step file. Can you check to confirm this please?